### DISCUSSION TITLE
Exception When Trying to Serialize

### DISCUSSION THREAD

**Discussion by seanocali** [k-paxian/dart-json-mapper#200] [Q&A] [answered]
Hi I'm new to Flutter/Dart coming from dotnet, first hiccup I ran into is json requiring a lot of work compared to C# JSON.net. Your plugin looks like the best option. Followed the documentation and ran this:

```dart
import 'package:dart_json_mapper/dart_json_mapper.dart';
import 'main.mapper.g.dart';

@jsonSerializable
class MyTestClass{
  String? name;
  int? age;
}

void main() {
  var myTestObj = MyTestClass();
  myTestObj.name = "Jesse Doe";
  myTestObj.age = 20;

  String serialized = JsonMapper.serialize(myTestObj);
}
```
And it throws this:

> Bad state: Reflectable has not been initialized.
> Please make sure that the first action taken by your program
> in `main` is to call `initializeReflectable()`.

**Comment by k-paxian** [comment, marked as answer]
It seems you are missing this part from your code
![image](https://user-images.githubusercontent.com/18071010/198237326-6dc16dcb-a1e3-4925-8372-979eb02b98c8.png)
https://github.com/k-paxian/dart-json-mapper#basic-setup
